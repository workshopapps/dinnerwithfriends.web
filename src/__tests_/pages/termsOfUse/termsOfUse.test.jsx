import { render } from '@testing-library/react';
import TermsOfUse from '../../../pages/termsOfUse/index.js';
import { BrowserRouter } from 'react-router-dom';

describe('Terms of use page', () => {
  test('expect terms of use page to render correctly', () => {
    // eslint-disable-next-line testing-library/render-result-naming-convention
    const signup = render(
        <BrowserRouter>
          <TermsOfUse />
        </BrowserRouter>
    );
    expect(TermsOfUse).toMatchSnapshot();
  });
});