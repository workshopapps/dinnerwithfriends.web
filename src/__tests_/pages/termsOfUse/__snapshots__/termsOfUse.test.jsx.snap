// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms of use page expect terms of use page to render correctly 1`] = `[Function]`;
